### PR TITLE
lang: Complement more languages text

### PR DESCRIPTION
**Pull request by Croden1999**
# About
- lang: Complement more languages text
  based on psvita lang, and fixed a few typo.
- gui/lang: Add a few lang strings.
- gui/settings dialog: Adjust the text display.

used [CXML-Decompiler](https://bitbucket.org/SilicaAndPina/cxml-decompiler) by SilicaAndPina dumped the original lang xml from psvita fw rco files, complement to `dialog`, `indicator` and `vita3k_update`. not exactly 100% correct for `vita3k_update`, i currently only understand en/zh-s/zh-t. need devs, contributors and users check this lang pr.

### CODE REVIEW

File: lang/ru.xml [hunk 1]
```diff
@@ -443,4 +443,19 @@ depending on its size and your hardware.</app_delete_note>
 		<confirm>Подтвердить</confirm>
 		<automatic_user_login>Автоматический вход пользователя</automatic_user_login>
 	</user_management>
+
+	<vita3k_update name="Обновление Vita3K">
+ 		<new_version_available>Имеется новая версия Vita3K.</new_version_available>
+		<back>Назад</back>
+		<downloading>Идет загрузка...
+После завершения копирования Vita3K автоматически перезапустится, а затем установит новое Vita3K.</downloading>
```
Comment (bookmist) [inline lang/ru.xml:451]: После завершения загрузки Vita3K автоматически перезапустится и установит новую версию.

File: lang/ru.xml [hunk 2]
```diff
@@ -443,4 +443,19 @@ depending on its size and your hardware.</app_delete_note>
...
+		<not_complete_update>Не удалось завершить обновление.</not_complete_update>
+		<next>Далее</next>
+		<update_vita3k>Обновить Vita3K?</update_vita3k>
+		<later_version_already_installed>Более поздняя версия Vita3K уже установлена.</later_version_already_installed>
+		<latest_version_already_installed>Установлена самая последняя версия Vita3K.</latest_version_already_installed>
```
Comment (bookmist) [inline lang/ru.xml:456]: Установлена последняя версия Vita3K.

**Review by bookmist — COMMENTED** [review]
Very good. No changes really needed, but I think some texts could be made a bit more clear.

**Comment by Croden1999** [comment]
> Very good. No changes really needed, but I think some texts could be made a bit more clear.

just add lang from psvita lang, only part of it needs to be modified

**Comment by nishinji** [comment]
You forgot to fix _Exit Manual Help_.
Your Japanese looks correct. Thank you!

File: lang/fr.xml [hunk 3]
```diff
@@ -321,7 +321,7 @@ Votre système Vita3K est prêt !</completed_setup>
 			<browse_app>Browse in app list</browse_app>
 			<browse_app_control>D-pad, Left Stick, Wheel in Up/Down or using Slider</browse_app_control>
 			<start_app>Start App</start_app>
```
Comment (Zangetsu38) [inline lang/fr.xml:323]:
```suggestion
			<start_app>Démarrer App</start_app>
```

File: lang/fr.xml [hunk 4]
```diff
@@ -321,7 +321,7 @@ Votre système Vita3K est prêt !</completed_setup>
...
-			<start_app_control>Wheel in Up/Down or using Slider</start_app_control>
+			<start_app_control>Click on Start or Press on Cross</start_app_control>
```
Comment (Zangetsu38) [inline lang/fr.xml:324]:
```suggestion
			<start_app_control>Cliquer sur Démarrer ou Appuyer sur Croix</start_app_control>
```

**Comment by Croden1999** [comment]
> You forgot to fix _Exit Manual Help_. Your Japanese looks correct. Thank you!

oh,yes, thank you for reminding me

File: lang/fr.xml (hunk 4 above)
Comment (Croden1999) [reply lang/fr.xml:324]: got it, i will push next time

**Comment by Croden1999** [comment]
everyone can review this pr, if someone need change, please tell me

File: lang/en-gb.xml [hunk 5]
```diff
@@ -83,7 +83,6 @@ Error code: {}</an_error_occurred>
 		<delete>Delete</delete>
 		<file_corrupted>The file is corrupt.</file_corrupted>
 		<microphone_disabled>Enable the microphone.</microphone_disabled>
-		<nospace>There is not enough free space on the memory card.</nospace>
```
Comment (Zangetsu38) [inline lang/en-gb.xml:86]: Why delete ?

Comment (Croden1999) [reply lang/en-gb.xml:86]: > Why delete ?

is repeated

File: vita3k/modules/SceCommonDialog/SceCommonDialog.cpp [hunk 6]
```diff
@@ -352,7 +353,7 @@ EXPORT(int, sceMsgDialogInit, const Ptr<SceMsgDialogParam> param) {
             emuenv.common_dialog.msg.btn_num = 0;
             break;
         case SCE_MSG_DIALOG_SYSMSG_TYPE_NOSPACE_CONTINUABLE:
-            emuenv.common_dialog.msg.message = common["nospace"];
+            emuenv.common_dialog.msg.message = save["not_free_space"];
```
Comment (Croden1999) [inline vita3k/modules/SceCommonDialog/SceCommonDialog.cpp:356]: so i changed to use 'save_data.save.not_free_space'

File: lang/fr.xml [hunk 7]
```diff
@@ -453,6 +453,8 @@ Une fois le téléchargement terminé, Vita3K redémarrera automatiquement et in
 		<later_version_already_installed>Une version plus récente de Vita3K est déjà installée.</later_version_already_installed>
 		<latest_version_already_installed>La dernière version de Vita3K est déjà installée.</latest_version_already_installed>
 		<new_features>Nouveautés de la version {}</new_features>
+		<author>Author</author>
+		<comments>Comments</comments>
```
Comment (Zangetsu38) [inline lang/fr.xml:457]:
```suggestion
		<comments>Commentaires</comments>
```

File: lang/fr.xml [hunk 8]
```diff
@@ -453,6 +453,8 @@ Une fois le téléchargement terminé, Vita3K redémarrera automatiquement et in
 		<later_version_already_installed>Une version plus récente de Vita3K est déjà installée.</later_version_already_installed>
 		<latest_version_already_installed>La dernière version de Vita3K est déjà installée.</latest_version_already_installed>
 		<new_features>Nouveautés de la version {}</new_features>
+		<author>Author</author>
```
Comment (Zangetsu38) [inline lang/fr.xml:456]:
```suggestion
		<authors>Auteurs</authors>
```

Comment (Zangetsu38) [reply lang/fr.xml:456]: I have write error in code i jyts see here, is Authors* need change it 😅

Comment (Croden1999) [reply lang/fr.xml:456]: okay, i got it

Comment (Croden1999) [reply lang/fr.xml:456]: okay, i got it

**Comment by Croden1999** [comment]
ready to review

**Review by Zangetsu38 — APPROVED** [review]

**Pull request merged**